articles/quickstart/spa/angular2/_includes/_centralized_login.md

```diff
@@ -27,11 +28,11 @@ import * as auth0 from 'auth0-js';
 export class AuthService {
 
   auth0 = new auth0.WebAuth({
-    clientID: '${account.clientId}',
-    domain: '${account.namespace}',
+    clientID: <YOUR AUTH0 CLIENT ID>,
+    domain: <YOUR AUTH0 DOMAIN>,
     responseType: 'token id_token',
-    audience: 'https://${account.namespace}/userinfo',
-    redirectUri: 'http://localhost:3000/callback',
+    audience: `https://${<YOUR AUTH0 DOMAIN>}/userinfo`,
+    redirectUri: <YOUR AUTH0 ALLOWED CALLBACK URL>,
     scope: 'openid'
   });
 
```

Comment:
Please use '${account.clientId}' and '${account.namespace}' as it will be replaced with the proper values.

For redirectUri please use http://localhost:3000/callback' and add a comment saying 'If you specified another callback URL please use that one here'.

Remove the 'scope' parameter as it will default to 'openid profile email'.

Remove the 'audience' parameter as it's not needed. 

Some of these changes are not done in other QS but we'll start updating them in that way. Please also make the changes in the sample code.

Reply:
Done for the doc. Need to update the source code.